packages/components/src/button/style.scss

```diff
@@ -22,6 +24,10 @@
 	padding: 7px 14px 9px;
 	appearance: none;
 
+	.rtl & {
```

Comment:
The `calypso` reset has some RTL styles but since the consumer isn't importing the `calypso` reset, we need to define them on a per-component basis.

Reply:
Let's use the `html[dir='rtl']` selector instead of `.rtl`. That's more standard and reliable.

Reply:
👍 I agree but I'm not aware of what might break by changing it (its in the base styles already)

Reply:
I don't expect anything to break: the change looks very safe.